recipe/meta.yaml

```diff
@@ -12,13 +12,21 @@ source:
   sha256: d71aab489b5288a6c96e9f9a7c9e13c54774dcb55a99a40e6cd1aca8be9ef3e6
 
 build:
-  number: 2
+  number: 3
   string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
   string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
   skip: true  # [win]
   script:
     - export FORCE_CUDA="1"    # [linux64 and cuda_compiler_version not in (undefined, 'None')]
```

Comment:
Are you sure you don't need to specify the supported GPUs like we do in the pytorch recipe.

Reply:
In this particular version something (not sure where) looks after that. In the newer version, I think we need to specify the supported GPUs: https://github.com/conda-forge/pytorch_scatter-feedstock/pull/44#issuecomment-1074434491

Reply:
ok. maybe you can just merge both PRs. I'm going to do the bad thing and just merge the OSX fixup. It should be ready in a few hours. You'll still need the `sed` hack for now (unless yosu want to build cuda)